packages/playwright-core/src/server/dispatchers/electronDispatcher.ts

```diff
@@ -67,6 +81,15 @@ export class ElectronApplicationDispatcher extends Dispatcher<ElectronApplicatio
     return { handle: ElementHandleDispatcher.fromJSHandle(this, result) };
   }
 
+  async updateSubscription(params: channels.ElectronApplicationUpdateSubscriptionParams): Promise<void> {
+    if (params.enabled)
+      this._subscriptions.add(params.event);
+    else
+      this._subscriptions.delete(params.event);
+    if (params.event === 'console' && params.enabled)
+      this._electronApplication._emitBufferedConsoleMessages();
```

Comment:
Why buffer for electron? IIRC, we do not buffer for pages/contexts.

Reply:
The Problem is that we get the `Runtime.consoleAPICalled` events before the `ElectronApplicationDispatcher` has been created in such a case. Moving the buffer emit logic into the constructor of ElectronApplicationDispatcher is also not working, since we don't know at that time if the client object has been created already.

I'm not a huge fan of dropping these ConsoleMessages on the floor, so buffering felt appropriate.

It's about this test: https://github.com/microsoft/playwright/blob/d3721631e3ff8f1feea4481bc4019da3b74ed7fc/tests/electron/electron-app.spec.ts#L77-L97